api/server.go

```diff
@@ -1,9 +1,5 @@
 package api
 
-// LICENSE: Apache
-// This is taken from https://github.com/philips/grpc-gateway-example/
```

Comment:
let's keep the attribution `This is taken from https://github.com/philips/grpc-gateway-example/` unless we are not using the code we copied anymore